### ISSUE TITLE
Autostart items not showing in systemtray

### ISSUE THREAD

**Issue by map7** [ch11ng/exwm#157]
I'm running exwm 0.4 on emacs 24.5.1 and when I start exwm it loads the system tray but the items which I've autostarted (like volti and nm-applet) are not displayed in the systemtray. If I start an application which uses the systemtray after I'm loaded then I can see that icon in the systemtray.

Here is my .xinitrc file
```
# Disable access control
xhost +SI:localuser:$USER

# Themes, etc
# gnome-settings-daemon &

# Fallback cursor
xsetroot -cursor_name left_ptr

# Keyboard repeat rate
#xset r rate 180 40

# Set capslock as ctrl
setxkbmap -layout us -option ctrl:nocaps

# If Emacs is started in server mode, `emacsclient` is a convenient way to edit
# files in place (used by e.g. `git commit`)
export VISUAL=emacsclient
export EDITOR="$VISUAL"

# # Start Emacs
# exec dbus-launch --exit-with-session emacs


xset b off &
xfce4-power-manager &
xscreensaver -no-splash &
nm-applet &                     # Network Manager
volti &                         # Volume manager
xmodmap -e 'keycode 135 = Super_R' &
xhost +
exec dbus-launch --exit-with-session emacs
```

Here is my exwm setup in emacs
```
;; Add paths (not required if EXWM is installed from GNU ELPA)
(add-to-list 'load-path "/home/map7/.emacs.d/elpa/xelb-0.6/")
(add-to-list 'load-path "/home/map7/.emacs.d/elpa/exwm-0.4/")
(add-to-list 'load-path "/home/map7/.emacs.d/elpa/cl-generic-0.2/")

;; Enable workspace
(setq exwm-workspace-number 10)

;; RandR (multi-screen support)
(require 'exwm-randr)
(setq exwm-randr-workspace-output-plist '(0 "VGA1"))
(add-hook 'exwm-randr-screen-change-hook
          (lambda ()
            (start-process-shell-command
             "xrandr" nil "xrandr --output VGA1 --right-of LVDS1 --auto")))
(exwm-randr-enable)

;; ;; Start server - helps with opening files
;; (server-start)


;; Load EXWM
(require 'exwm)
(require 'exwm-config)
(exwm-config-default)

;; + 'slock' is a simple X display locker provided by suckless tools. 'i3lock'
;;   is a more feature-rich alternative.
(exwm-input-set-key (kbd "s-<f2>")
                    (lambda () (interactive) (start-process "" nil "slock")))

;; System tray
(require 'exwm-systemtray)
(exwm-systemtray-enable)
```

**Comment by map7** [comment]
Current work around I'm using is to use trayer

```
xset b off &
xfce4-power-manager &
xscreensaver -no-splash &
nm-applet &                     # Network Manager
volti &                         # Volume manager
xmodmap -e 'keycode 135 = Super_R' &
xhost +
exec dbus-launch --exit-with-session emacs &
sleep 10
trayer --edge bottom --align right --width 10 --height 20 --transparent true --alpha 0 --tint 0x00000000
```

It's not the best solution

**Comment by ch11ng** [comment]
Fixed in ch11ng/xelb@ecd68de3176291afcf377ab9465491947f1aacac and 009854e766728ed17741bf9f7e6d86f9ccc71871.
